src/commands/app/init.js

```diff
@@ -38,20 +38,30 @@ class InitCommand extends BaseCommand {
     let services = 'AdobeTargetSDK,AdobeAnalyticsSDK,CampaignSDK,McDataServicesSdk,AudienceManagerCustomerSDK' // todo fetch those from console when no --import
 
     if (!(flags.import || flags.yes)) {
-      const { accessToken, env: imsEnv } = await getCliInfo()
-
+      let token
+      let cliEnv
       try {
-        const generatedFile = 'console.json'
-        env.register(require.resolve('@adobe/generator-aio-console'), 'gen-console')
-        res = await env.run('gen-console', {
-          'destination-file': generatedFile,
-          'access-token': accessToken,
-          'ims-env': imsEnv
-        })
-        // trigger import
-        flags.import = generatedFile
+        const { accessToken, env: imsEnv } = await getCliInfo()
```

Comment:
I'm confused why you just didn't take this line and just put in the bottom try catch, without resulting in this extra code. Having a custom error message is irrelevant.

Reply:
OK. I thought in this case it would be good to log custom error. Anyway I can move this to the other try catch block